package.json

```diff
@@ -4,8 +4,8 @@
   "description": "A rural hospital information management system.",
   "main": "npm run build",
   "scripts": {
-    "app": "NODE_ENV=production ./node_modules/.bin/gulp build && cd bin && NODE_ENV=production node server/app.js",
-    "dev": "NODE_ENV=development ./node_modules/.bin/gulp build && cd bin && NODE_ENV=development node server/app.js",
+    "app": "cross-env gulp build && cd bin && set NODE_ENV=production && node server/app.js",
```

Comment:
Could you put `NODE_ENV=production` in front of the gulp build here?  That way it will minify the code using [this line in the gulpfile](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/gulpfile.js#L38) which activates the minification [here](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/gulpfile.js#L174).